### ISSUE TITLE
Focusable not disabled when `display: none`

### ISSUE THREAD

**Issue by rozele** [Microsoft/react-native-windows#1705]
### Environment

1. `react-native -v`: *
2. `npm ls rnpm-plugin-windows`: *
3. `npm ls react-native-windows`: *
4. `node -v`: *
5. `npm -v`: *
6. `yarn --version`: *

Then, specify:

- Target Platform: UWP

- Target Platform Version(s): UWP *

- Target Device(s): *

- Development Operating System: Windows

- Visual Studio Version: 2017

### Steps to Reproduce

(Write your steps here:)

1. Copy repro below into index.windows.js in Playground
2. Run playground
3. Click the `Hide` button

### Expected Behavior

* When you press `Tab`, it should only have one tab stop to cycle over, the Button.
* When you press `Tab` and the tab stop is on the Button, the border highlight should match the dimensions of the Button, but it does not.

### Actual Behavior

![ezgif-3-3508068807](https://user-images.githubusercontent.com/1106239/37209128-06dcf584-2372-11e8-818d-ab4a64df965e.gif)

### Reproducible Demo

```
class Playground extends Component {
  state = {
    display: true
  }

  render() {
    return (
      <View style={styles.container}>
        <FocusableView 
          isTabStop={true}
          style={{display: this.state.display ? 'flex' : 'none'}}>
          <Text>Hello</Text>
        </FocusableView>
        <Button 
          onPress={() => this.setState({display: !this.state.display})}
          title={this.state.display ? 'Hide' : 'Show'} 
        />
      </View>
    );
  }
}
```

**Comment by rozele** [comment]
cc @reseul

**Comment by rozele** [comment]
Sounds like this was resolved by #1707

**Comment by rozele** [comment]
Confirmed closed.
